### ISSUE TITLE
already trained ResNet101 Model

### ISSUE THREAD

**Issue by mkgmxgit** [vijayg15/Keras-MultiClass-Image-Classification#1]
Hello,
thank you very much for providing the code and for the very interesting work.I read your thesis and wanted to ask if you would like to provide the best working already trained model (ResNet101 + Data augmentation + Dense Layers + Dropout + Batch Normalization) in the download area. At the moment it is not listed there as far as I have seen.
Thank you very much in advance and many greetings
mk
